data/layout/telescope_positions-North-1LST.ecsv

```diff
@@ -52,4 +52,4 @@
 #       value: 3.0
 # schema: astropy-2.0
 telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
-L-01 0.0 0.0 0.0 28.762166100000005 -17.8920302 217609.2270142641 3185067.278324084 2142.0
+L-01 0.0 0.0 0.0 28.762158000000007 -17.891571 217654.06411838497 3185065.2899643113 2142.0
```

Comment:
Not really relevant, as this is only an example. But L-01 position is different to what the CTAO list says:

https://gitlab.cta-observatory.org/cta-science/cta-science/-/blob/master/Array%20Element%20Positions/CTAN_ArrayElements_Positions.md

Reply:
Good to know. I just took the data from your telescope-layouts repo.
I will create an issue to validate the layout positions.

Reply:
OK. Not so important as I assume that this will all go into the database.